.golangci.yml

```diff
@@ -0,0 +1,10 @@
+linters:
```

Comment:
`golangci-lint run` fails on Windows with standard Git configuration due to CRLFs. See https://github.com/golangci/golangci-lint/issues/580. Forcing Git to use LF for Go files even on Windows is apparently the only fix, which is unfortunate, but it is something I've seen before in Go repos so I suppose people are used to it.